### PR TITLE
Predicted NTs

### PR DESCRIPTION
**Pull request by Clare72**
function to get nt predictions
ability to use predicted nts to find downstream ntrs
fixes to get_nt_receptors_in_downstream_neurons: only exact match cluster types, adding Serotonin to dictionary, dealing with lists in downstream_class

### CODE REVIEW

**Review by Robbie1977 — APPROVED** [review]

File: src/vfb_connect/cross_server_tools.py [hunk 1]
```diff
@@ -977,39 +984,106 @@ def cypher_query(self, query, return_dataframe=True, verbose=False):
             return pd.DataFrame.from_records(dc)
         return dc
 
-    def get_nt_receptors_in_downstream_neurons(self, upstream_type, downstream_type='neuron', weight=0 , return_dataframe=True, verbose=False):
+    def get_nt_predictions(self, term, verbose=False):
```
Comment (Robbie1977) [inline src/vfb_connect/cross_server_tools.py:987]: Do we want this added as a property of the neuron or is the existing N.capable_of and X.related_terms enough?

Comment (Clare72) [reply src/vfb_connect/cross_server_tools.py:987]: Might be good to have a more specific, explicitly-named property

Comment (Robbie1977) [reply src/vfb_connect/cross_server_tools.py:987]: neurotransmitter and neurotransmitter_prediction? (based on the Capable_of having a confidence score)?

Comment (Robbie1977) [reply src/vfb_connect/cross_server_tools.py:987]: we can filter by the same list just don't like to add more query time to term loading so easier if we can use the data we have

Comment (Clare72) [reply src/vfb_connect/cross_server_tools.py:987]: maybe do neurotransmitter_predictions for individuals and classes (will need to be a list if using at the class level) and only get known_neurotransmitters for classes (also needs to be a list)

Comment (Clare72) [reply src/vfb_connect/cross_server_tools.py:987]: nb predictions will be the GO terms and known will be the neo labels, unless you want to do some kind of mapping.

Comment (Robbie1977) [reply src/vfb_connect/cross_server_tools.py:987]: I'll get this merged for now anyway

**Pull request merged**